src/GogeDao.sol

```diff
@@ -370,718 +476,955 @@ contract GogeDAO is Ownable {
-            }
-            else if (pollTypes[_pollNum] == PollType.updateGasForProcessing) {
-                UpdateGasForProcessing memory updateGasForProcessing;
-                (,updateGasForProcessing,) = getUpdateGasForProcessing(_pollNum);
-                IGogeERC20(governanceTokenAddr).updateGasForProcessing(updateGasForProcessing.amount);
-            }
-            else if (pollTypes[_pollNum] == PollType.updateMinimumBalanceForDividends) {
-                UpdateMinimumBalanceForDividends memory updateMinimumBalanceForDividends;
-                (,updateMinimumBalanceForDividends,) = getUpdateMinimumBalanceForDividends(_pollNum);
-                IGogeERC20(governanceTokenAddr).updateMinimumBalanceForDividends(updateMinimumBalanceForDividends.amount);
-            }
-            else if (pollTypes[_pollNum] == PollType.modifyBlacklist) {
-                ModifyBlacklist memory modifyBlacklist;
-                (,modifyBlacklist,) = getModifyBlacklist(_pollNum);
-                IGogeERC20(governanceTokenAddr).modifyBlacklist(modifyBlacklist.addr, modifyBlacklist.blacklisted);
-            }
-            else if (pollTypes[_pollNum] == PollType.transferOwnership) {
-                TransferOwnership memory transferOwnership;
-                (,transferOwnership,) = getTransferOwnership(_pollNum);
-                IGogeERC20(governanceTokenAddr)._transferOwnership(transferOwnership.addr);
-            }
-            else if (pollTypes[_pollNum] == PollType.migrateTreasury) {
-                MigrateTreasury memory migrateTreasury;
-                (,migrateTreasury,) = getMigrateTreasury(_pollNum);
-                IGogeERC20(migrateTreasury.token).transfer(migrateTreasury.addr, IGogeERC20(migrateTreasury.token).balanceOf(address(this)));
-            }
-            else if (pollTypes[_pollNum] == PollType.setQuorum) {
-                SetQuorum memory setQuorum;
-                (,setQuorum,) = getSetQuorum(_pollNum);
-                updateQuorum(setQuorum.amount);
-            }
-            else if (pollTypes[_pollNum] == PollType.setMinPollPeriod) {
-                SetMinPollPeriod memory setMinPollPeriod;
-                (,setMinPollPeriod,) = getSetMinPollPeriod(_pollNum);
-                updateMinPollPeriod(setMinPollPeriod.amount);
-            }
-            else if (pollTypes[_pollNum] == PollType.updateGovernanceToken) {
-                UpdateGovernanceToken memory updateGovernanceToken;
-                (,updateGovernanceToken,) = getUpdateGovernanceToken(_pollNum);
-                changeGovernanceToken(updateGovernanceToken.addr);
+        if (l > 0) {
+            for(uint256 i = 0; i < l; i++) {
+
+                (bool sent,) = teamMembers[i].call{value: amount}("");
+                require(sent, "Failed to pay team");
+
+                teamBalance -= amount;
             }
+        }
+
+        (bool sent,) = teamMembers[l].call{value: teamBalance}("");
```

Comment:
The issue is when we calculate amount, the reason why we send the remaining balance after the loop to teamMembers[len] is to send the remaining balance which could be amount-1 (because of overflow). Also added a require to the beginning of payTeam to ensure teamMembers.length was greater than 0 verifying there exists team members